### PR TITLE
[Documentation][zos_operator_action_query] Add and standarize docstrings on modules/zos_operator_action_query.py

### PR DESCRIPTION
**Pull request by IsaacVRey**
##### SUMMARY

Update docstrings on zos_operator_action_query to the numpy standard
This addresses part of #1316 

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
zos_operator_action_query.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by richp405 — APPROVED** [review]

**Review by fernandofloresg — APPROVED** [review]

**Comment by fernandofloresg** [comment]
Do not merge until #1388 is merged :)

**Pull request merged**
